### ISSUE TITLE
JDK 8: "java.lang.NoClassDefFoundError" on creating a page with javadoc

### ISSUE THREAD

**Issue by anarchuser** [Kotlin/dokka#790]
**Describe the bug**
While trying to create javadoc documentation for a java&kotlin project with JDK v8, the execution fails with a `com/sun/javadoc/DocErrorReporter` error being thrown. 

**Expected behaviour**
This has worked for other people so already and should produce javadoc documentation for the whole project.

**Stacktrace**
```
Caused by: java.lang.NoClassDefFoundError: com/sun/javadoc/DocErrorReporter
        at org.jetbrains.dokka.javadoc.JavadocGenerator.buildPages(dokka-adapters.kt:16)
        at org.jetbrains.dokka.GeneratorKt.buildAll(Generator.kt:13)
        at org.jetbrains.dokka.GeneratorKt.buildAll(Generator.kt:23)
        at org.jetbrains.dokka.DokkaGenerator.generate(DokkaGenerator.kt:47)
...
```

**Installation**
- Operating system: Arch Linux 5.5.13
- Build tool: Gradle v6.0.1
- Dokka version: 0.9.17
- Kotlin version: 1.3.50

**Comment by Proximyst** [comment]
This is also present in dokka version `org.jetbrains.dokka` (Gradle), Kotlin version 1.3.72, Gradle 6.3 running on OpenJDK 14.0.1. I'm running Void Linux.

**Comment by kamildoleglo** [comment]
@anarchuser Could you please bump to the newest dokka version possible and see if the problem still exists?


@Proximyst Javadoc format does not work with JDK > 8 for now, this will be fixed in 0.11.0

**Comment by kamildoleglo** [comment]
Fixed in 1.4.10.2
